src/pkgdb/pkg-query.cc

```diff
@@ -66,9 +66,7 @@ PkgQueryArgs::InvalidPkgQueryArgException::errorMessage(
       case PQEC_INVALID_MATCH_STYLE:
         return "Query `matchStyle' must be set when `match' is used";
         break;
-      default:
-        return "Unexpected PkgQuery error";
-        break;
+      default: return "Unexpected PkgQuery error"; break;
```

Comment:
style(non-blocking): Bothers me a bit that this is all on one line, compared to the above cases.

Reply:
I think the formatter did that, I don't remember editing that line.
I'll rerun it and see.